src/mmw/js/src/analyze/templates/tableRow.html

```diff
@@ -1,3 +1,3 @@
 <td>{{ type }}</td>
-<td class="strong text-right">{{ scaledArea|round(2)|toLocaleString }}</td>
+<td class="strong text-right">{{ scaledArea|round(2)|toLocaleString|pad(2) }}</td>
```

Comment:
[`toLocaleString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) also takes optional parameters which tell it to display a certain number of `minimumFractionDigits`. It may be simpler to modify the [`toLocaleString` filter](https://github.com/WikiWatershed/model-my-watershed/blob/feb1affef89f412d038ff47385851bb7c5ad38d1/src/mmw/js/src/core/filters.js#L7-L9) to take an optional parameter which does the parsing for us.